docs/legacy/configuring.asciidoc

```diff
@@ -73,8 +72,6 @@ include::{libbeat-dir}/shared-ssl-config.asciidoc[]
 include::ssl-input-settings.asciidoc[]
 // END SSL SECTION --------------------------------------------
 
-include::./transaction-metrics.asciidoc[]
```

Comment:
**issue**: this might require coordinating with the kibana team to remove the companion page on their documentation (should be a config option): https://github.com/elastic/kibana/pull/82379

Reply:
Yup. This will need to be removed and backported from the Kibana docs before docs ci will pass on this PR (as that content links to this content). I'll open a PR in the Kibana repo take care of that.

Reply:
Opened https://github.com/elastic/kibana/pull/149896

Reply:
Merged and backported in the Kibana repo. Rerunning docs ci now...